code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -133,14 +131,21 @@
 	amount_per_transfer_from_this = 15
 	volume = 15
 	flags = FPRINT
-
-/obj/item/weapon/reagent_containers/hypospray/autoinjector/paralytic_injector/New()
-	..()
-	reagents.remove_reagent(DOCTORSDELIGHT, 30)
-	reagents.add_reagent(SUX, 10)
+	refill_reagent_list = list(SUX = 10)
```

Comment:
why 10 if your volume is 15?

Reply:
Holdover from the base type I guess, I'll clean it up.